serverless/pages/saved-objects.mdx

```diff
@@ -85,3 +85,16 @@ TBD: Are these settings configurable in serverless?
 The <DocLink id="enKibanaSettings" section="savedObjects-maxImportExportSize">`savedObjects.maxImportExportSize`</DocLink> configuration setting limits the number of saved objects that you can export.
 </DocCallOut> */}
 
+## Copy to other spaces
+
+Copy saved objects and their related objects between spaces.
+
+1. Click the actions icon <DocIcon type="boxesHorizontal" title="Actions" />.
+1. Click **Copy to spaces**.
+1. Specify whether to automatically overwrite any objects that already exist
+in the target spaces, or resolve them on a per-object basis.
+1. Select the spaces in which to copy the object.
```

Comment:
```suggestion
2. Click **Copy to spaces**.
3. Specify whether to automatically overwrite any objects that already exist
in the target spaces, or resolve them on a per-object basis.
4. Select the spaces in which to copy the object.
```

Reply:
@jeramysoucy this markup doesn't matter in MDX, it just indicates it's an ordered list. And It's easier to maintain if we don't update the numbers manually in case the instructions change in the future